third_party/clang/iOS.cmake

```diff
@@ -15,7 +15,7 @@ set(DIST_ARCH ${IOS_ARCH})
 
 add_definitions(-DIOS)
 
-set(IOS_MIN_TARGET "11.0")
+set(IOS_MIN_TARGET "14.0")
```

Comment:
Was this necessary? Why?

Reply:
Apparently this is not needed anymore. Reverted in 02830cd